lib/bundle.js

```diff
@@ -103,31 +199,33 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     return systemBuilder.trace(firstModule, cfg);
```

Comment:
This doesn't seem right - can we not glob prior to the trace operation and expand the glob into the arithmetic?